docs/workflow/grid_surrogates.rst

```diff
@@ -1,4 +1,97 @@
 Surrogate Models for Grid Outcomes
 ==================================
 
-The DISPATCHES...
+
+1. The Steady-State Co-Optimization with Market Interactions
+-------------------------------------------------------------
+
+We mainly developed three surrogate models. Each surrogate takes 8 inputs from the production cost model (PCM) Prescient
```

Comment:
Replace "We mainly developed three surrogate models" with "We developed two surrogate model architectures to map market input parameters to market outputs".